src/containers/PageBuilder/Primitives/Image/Image.js

```diff
@@ -41,17 +42,22 @@ export const FieldImage = React.forwardRef((props, ref) => {
   const { width: aspectWidth, height: aspectHeight } = firstImageVariant || {};
 
   const classes = classNames(rootClassName || css.fieldImage, className);
+  const imageLinkHref = props.link?.href || null;
+
+  const responsiveImage = (
+    <ResponsiveImage
+      className={css.fieldImage}
+      ref={ref}
+      alt={alt}
+      image={image}
+      variants={variantNames}
+      sizes={sizes}
+      {...otherProps}
+    />
+  );
   return (
     <AspectRatioWrapper className={classes} width={aspectWidth || 1} height={aspectHeight || 1}>
-      <ResponsiveImage
-        className={css.fieldImage}
-        ref={ref}
-        alt={alt}
-        image={image}
-        variants={variantNames}
-        sizes={sizes}
-        {...otherProps}
-      />
+      {imageLinkHref ? <Link href={imageLinkHref}>{responsiveImage}</Link> : responsiveImage}
```

Comment:
Perhaps you could do something like this:
```suggestion
      {imageLinkHref ? <Link href={imageLinkHref} title={alt}>{responsiveImage}</Link> : responsiveImage}
```

When anchor is wrapping an image, the **_alt_** attribute of the image is not selected to be used as "tooltip". Instead, the anchor's content is used, but as that's an img, it won't show anything.

![Screenshot 2024-10-07 at 19 00 31](https://github.com/user-attachments/assets/33b579ea-4ff8-4da2-aa46-d8591c26c33a)